cyclops-ui/src/components/pages/Modules/Modules.tsx

```diff
@@ -26,6 +28,9 @@ const Modules = () => {
   const [allData, setAllData] = useState([]);
   const [filteredData, setFilteredData] = useState([]);
   const [loadingModules, setLoadingModules] = useState(false);
+  const [displaymodule, setDisplayModule] = useState<string[]>([]);
+  const resourceFilter = ["Healty", "Unhealthy", "Unknown"];
+  const [isHovered, setIsHovered] = useState(false);
```

Comment:
You don't need this state. If you need to update the component style if the component is hovered, you can use CSS

Reply:
Removed state